packages/fhir-care-team/src/components/CreateEditCareTeam/utils.tsx

```diff
@@ -101,22 +101,19 @@ export const submitForm = async (
   };
 
   const serve = new FHIRServiceClass(fhirBaseURL, FHIR_CARE_TEAM);
+  let successNotifictaionMessage: string;
   if (id) {
-    await serve
-      .update(payload)
-      // TODO - possible place to use translation plurals
-      .then(() => sendSuccessNotification(t('Successfully Updated Care Teams')))
-      .catch(() => {
-        sendErrorNotification(t('An error occurred'));
-      });
+    successNotifictaionMessage = t('Successfully Updated Care Teams');
   } else {
-    await serve
-      .create(payload)
-      .then(() => sendSuccessNotification(t('Successfully Added Care Teams')))
-      .catch(() => {
-        sendErrorNotification(t('An error occurred'));
-      });
+    successNotifictaionMessage = t('Successfully Added Care Teams');
   }
```

Comment:
nitpick: we can get rid of the else here, if we initialize the `successNotificationMessage` with either of the values. i.e.
```typescript
  let successNotifictaionMessage = t('Successfully added care teams')
  if (id) { successNotifictaionMessage = t('Successfully updated care teams') }
```